lib/grape/validations/params_scope.rb

```diff
@@ -140,9 +140,14 @@ def require_optional_fields(context, opts)
       end
 
       def validate_attributes(attrs, opts, &block)
```

Comment:
Then I would have to split the options [here](https://github.com/ruby-grape/grape/blob/master/lib/grape/dsl/parameters.rb#L97), no?

Hmm, I guess I got a bit carried away with the future other options idea :)

In my opinion, the difference is that the rest of the `validations` refer to a specific validation type with options whereas `fail_fast` (and possible future validation options) influences the validation process across multiple validations.

But I could be wrong. I just dove into grape code for the first time :)